admin/category_product_listing.php

```diff
@@ -747,28 +747,28 @@
                     echo zen_icon('linked', IMAGE_ICON_LINKED, size: 'lg', hidden: true);
                   }
                   if ($category['categories_status'] === '1') { ?>
-                    <a href="<?php echo zen_href_link(FILENAME_CATEGORY_PRODUCT_LISTING, 'action=setflag_categories&flag=0&cID=' . $category['categories_id'] . '&cPath=' . $cPath . (isset($_GET['page']) ? '&page=' . $_GET['page'] : '') . $search_parameter); ?>" role="button">
-                      <?php echo zen_icon('enabled', IMAGE_ICON_STATUS_ON, size: 'lg', hidden: true); ?>
+                    <a href="<?= zen_href_link(FILENAME_CATEGORY_PRODUCT_LISTING, 'action=setflag_categories&flag=0&cID=' . $category['categories_id'] . '&cPath=' . $cPath . '&' . zen_get_all_get_params(['action', 'cID', 'cPath', 'flag',])) ?>" role="button">
```

Comment:
The more standard, albeit undocumented, usage of `zen_get_all_get_params` accompanied by additional URL parameters is to append the additional parameters to the function's output.  If `zen_get_all_get_params` finds non-excluded parameters, its return ends in `&`.

For this statement, for instance, use
```php
 zen_get_all_get_params(['action', 'cID', 'cPath', 'flag',]) . 'action=setflag_categories&flag=0&cID=' . $category['categories_id'] . '&cPath=' . $cPath
```

This same comment applies to all usages of `zen_get_all_get_params` in this PR.

Reply:
> ...albeit undocumented...

the horror....